buildtools/pipenv/pipenv.go

```diff
@@ -0,0 +1,156 @@
+package pipenv
+
+import (
+	"encoding/json"
+
+	"github.com/fossas/fossa-cli/errors"
+	"github.com/fossas/fossa-cli/exec"
+	"github.com/fossas/fossa-cli/graph"
+	"github.com/fossas/fossa-cli/pkg"
+)
+
+// Pipenv enables python.go to use a custom Deps() function for testing.
+type Pipenv interface {
+	Deps() (graph.Deps, error)
+}
+
+// Executable allows a custom DataAccess function to be used by Deps().
+type Executable interface {
+	DataAccess(string) (string, error)
+}
+
+// SystemPipenv implements Pipenv.
+type SystemPipenv struct {
```

Comment:
I'm still unsure about naming implementations of tools as `SystemFoo`. I think I'd prefer `FooCmd`.